src/style.css

```diff
@@ -0,0 +1,102 @@
+@import url(./default.css);
+
+@import url(./stars.css);
+
+@import url(./var.css);
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 198px;
+
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+}
+
+.card__product {
+  margin: 32px 0 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin: 0 0 4px;
+  width: 166px;
+
+  font-weight: 500;
+  line-height: 18px;
+  font-size: var(--min-font-size);
+  color: var(--font-dark-blue-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+  width: 166px;
+  text-align: left;
+
+  font-size: 10px;
+  color: rgb(97, 96, 112);
+  font-weight: var(--min-font-weight);
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-content: flex-end;
+  align-items: flex-end;
+  margin-bottom: 24px;
+  width: 166px;
+}
+
+.reviews_total {
+  font-size: 10px;
+  line-height: 15px;
+  font-weight: var(--min-font-weight);
+  color: var(--font-dark-blue-color);
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  width: 166px;
+
+  line-height: 18px;
+}
+
+.card__price {
+  font-size: var(--min-font-size);
+  color: var(--font-light-grey-color);
+}
+
+.card__amount {
+  font-size: 16px;
+  font-weight: var(--max-font-weight);
+  color: var(--font-dark-blue-color);
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 16px;
+
+  height: 38px;
+  font-size: 14px;
+  text-transform: uppercase;
+  width: 164px;
+  border-radius: 5px;
+  font-weight: var(--max-font-weight);
+  background-color: var(--buttton-light-blue);
+  color: rgb(255, 255, 255);
+  text-decoration: none;
```

Comment:
Group styles - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Fit it in all places